package.json

```diff
@@ -0,0 +1,39 @@
+{
+    "name": "osu-wiki",
+    "description": "home of the osu! wiki",
+    "repository": "github:ppy/osu-wiki",
+    "dependencies": {
+        "remark-cli": "latest",
+        "remark": "latest",
+        "remark-preset-lint-markdown-style-guide": "latest",
+        "remark-frontmatter": "latest",
+        "remark-lint-definition-case": "latest",
+        "remark-lint-fenced-code-flag": "latest",
+        "remark-lint-final-definition": "latest",
+        "remark-lint-final-newline": "latest",
+        "remark-lint-list-item-indent": "latest",
+        "remark-lint-list-item-spacing": "latest",
+        "remark-lint-maximum-heading-length": "latest",
+        "remark-lint-maximum-line-length": "latest",
+        "remark-lint-no-duplicate-definitions": "latest",
+        "remark-lint-no-duplicate-headings": "latest",
+        "remark-lint-no-duplicate-headings-in-section": "latest",
+        "remark-lint-no-emphasis-as-heading": "latest",
+        "remark-lint-no-file-name-articles": "latest",
+        "remark-lint-no-file-name-consecutive-dashes": "latest",
+        "remark-lint-no-file-name-irregular-characters": "latest",
+        "remark-lint-no-file-name-mixed-case": "latest",
+        "remark-lint-no-heading-content-indent": "latest",
+        "remark-lint-no-heading-indent": "latest",
+        "remark-lint-no-heading-like-paragraph": "latest",
+        "remark-lint-no-html": "latest",
+        "remark-lint-no-missing-blank-lines": "latest",
+        "remark-lint-no-shortcut-reference-link": "latest",
+        "remark-lint-no-tabs": "latest",
+        "remark-lint-no-undefined-references": "latest",
+        "remark-lint-no-unused-definitions": "latest",
+        "remark-lint-ordered-list-marker-value": "latest",
+        "remark-lint-table-pipe-alignment": "latest",
+        "vfile-reporter-position": "latest"
```

Comment:
should specify at least the major versions because you don't know if new major versions will break your Action. when you do `npm install <package>` from scratch it'll add something like `^1.0.0`, the `^` means that any 1.x.x is okay

or is there some reason you did want latest always  ?

Reply:
I figured we might not want to bother with tracking dependency versions, but seeing that npm is a ~~trash fire~~ minefield I'm thinking to go the complete opposite way and hard-lock dependency versions (including transitives, through the lock file) to what's latest right now and only update when it stops working.

I wouldn't want to add dependabot or something like that to this repository as it's going to be a source of unnecessary noise.

Reply:
It's considered bad practice not to pin versions. We might need to pin versions just in case the library changes its API one way or another.